operator/v1/types_network.go

```diff
@@ -533,6 +536,16 @@ type ProxyConfig struct {
 	ProxyArguments map[string]ProxyArgumentList `json:"proxyArguments,omitempty"`
 }
 
+// EgressIPConfig defines the configuration knobs for egressip
+type EgressIPConfig struct {
+	// ReachabilityTotalTimeout configures the EgressIP node reachability check timeout in seconds.
```

Comment:
It would be good to add a comment in here that setting the value overly large may cause slow convergence

Out of interest, why was it decided to set a total timeout rather than allowing configuration or timeout and max retries? Eg timeout 1 second, 9 retries, I can calculate my total as 9s

Reply:
Because it's about the total time that we give until we declare a node down and stop trying.

> timeout 1 second, 9 retries, I can calculate my total as 9s
Because that isn't true if the operation fails, but doesn't time out. 
In that case, 10 * operation(timeout 1 second) < 10 seconds

Reply:
> Because it's about the total time that we give until we declare a node down and stop trying.

As long as this is clearly written in the godoc I'm happy